src/main/java/seedu/address/logic/commands/AssetCommand.java

```diff
@@ -39,7 +39,7 @@ public class EditAssetCommand extends Command {
      * @param target Previous asset to replace.
      * @param editedAsset New asset to replace with.
      */
-    public EditAssetCommand(Asset target, Asset editedAsset) {
+    public AssetCommand(Asset target, Asset editedAsset) {
         requireNonNull(target);
         requireNonNull(editedAsset);
```

Comment:
Would it be better to use `requireAllNonNull`?

Reply:
Done